File: easy/1672RichestCustomer/1672Solution.c
int maximumWealth(int** accounts, int accountsSize, int* accountsColSize) {
    int res = 0;
    for(int costumer = 0; costumer < accountsSize; costumer++){
        int sum = 0;
        for(int wealth = 0; wealth < *accountsColSize; wealth++) sum += accounts[costumer][wealth];
       if(sum > res) res = sum;
    }
    return res;
}

File: easy/1672RichestCustomer/1672Solution.py
from typing import List
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        res = 0
        for customer in range(0, len(accounts)): 
            sum = 0
            for wealth in range(0,len(accounts[customer])):
                sum += accounts[customer][wealth]
            if(sum > res):
                res = sum
        return res
